Tests/LibWeb/Text/input/DOM/range-bounding-client-rect-with-nested-text.html

```diff
@@ -0,0 +1,47 @@
+<!--refer to https://wpt.live/css/cssom-view/range-bounding-client-rect-with-nested-text.html-->
```

Comment:
could we add a space before `>` in the end, so `--` is not automatically added when the link is clicked